src/App.tsx

```diff
@@ -1,61 +1,186 @@
 import './App.scss';
+import React, { useState } from 'react';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+
+import { User } from './Types/user';
+import { TodoList } from './components/TodoList';
+import { Todo } from './Types/todo';
+import { FormFields } from './Types/formFields';
+import { findUserByUserId } from './utils/fundUserByUserId';
+import { ErrorsFields } from './Types/errorsFields';
+
+const formFields: FormFields = {
+  title: '',
+  selectedUserId: '0',
+};
```

Comment:
- Store information about errors in the component, in the state.
- And actually, you don't show an error message if there is no user selected
<img width="474" alt="image" src="https://github.com/user-attachments/assets/2e78540a-b1d0-4390-a227-e2198fbb1cdb">